### DISCUSSION TITLE
How to change the information in api.github events/public?

### DISCUSSION THREAD

**Discussion by phucvq** [community/community#118489] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

I use GitHub to learn git, but unfortunately I use my company git account to push the example code to GitHub, then the company scan that information and force me to change the information in that or delete that information. But I permanently deleted my repo. The information in that api still exists, so what can I do?  
<img width="576" alt="Screenshot 2024-04-10 at 10 39 14" src="https://github.com/community/community/assets/7194593/62a5eeb5-89ce-4fce-82b1-d4484f2b4514">

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @phucvq ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
